docs/_docs/ruby-101.md

```diff
@@ -3,24 +3,27 @@ title: Ruby 101
 permalink: /docs/ruby-101/
 ---
 
-Jekyll is written in Ruby. If you're new to Ruby, this page is to help you get
-up to speed with some of the terminology.
+Jekyll is written in Ruby. If you're new to Ruby, this page helps you learn some of the terminology.
 
 ## Gems
 
-A gem is code you can include in Ruby projects. It allows you to package up functionality and share it across other projects or with other people. Gems can perform functionality such as:
+Gems are code you can include in Ruby projects. Gems package specific functionality. You can share gems across multiple projects or with other people. 
+Gems can perform actions like:
 
 * Converting a Ruby object to JSON
 * Pagination
 * Interacting with APIs such as GitHub
-* Jekyll itself is a gem as well as many Jekyll plugins including
+
+Jekyll is a gem. Many Jekyll [plugins]({{ '/docs/plugins/' | relative_url }}) are also gems, including
 [jekyll-feed](https://github.com/jekyll/jekyll-feed),
 [jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag) and
 [jekyll-archives](https://github.com/jekyll/jekyll-archives).
 
 ## Gemfile
 
-A `Gemfile` is a list of gems required for your site. For a simple Jekyll site it might look something like this:
+A `Gemfile` is a list of gems used by your site. Every Jekyll site has a Gemfile in the main folder. 
```

Comment:
In this chapter, does the use of different formatting (monospace "`Gemfile`" vs plain text "Gemfile") implies different meanings of this word?